src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -364,6 +360,7 @@ export function getStatusContents(statusType, details = {}, name = {}) {
               corrected for the following {pluralize.remand}
             </p>
             <ul>{remandIssues}</ul>
+            <p>The case is returned to the VBA to complete a "remand" order.</p>
```

Comment:
@gnakm should "remand", "remand order" or neither be in quotes here?

Reply:
"remand" being in quotes is correct.

Reply:
Thanks @gnakm.